libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -338,8 +340,8 @@ void AP_ESC_Telem::update()
     if (logger && logger->logging_enabled()) {
 
         for (uint8_t i = 0; i < ESC_TELEM_MAX_ESCS; i++) {
-            if ((_active_escs_mask & (1<<i)) &&  (_last_telem_data_ms[i] >  _last_telem_log_ms[i]
-                || _last_rpm_update_us[i] > _last_telem_log_ms[i] * 1000)) {
+            if ((_active_escs_mask & (1<<i)) &&  (_last_telem_data_ms[i] - _last_telem_log_ms[i] > 100
```

Comment:
Help me understand this change - update() is run at 10Hz from AP_Vehicle, so why do we need to check it again here?

Reply:
I assumed it was running at RCout rate. Isn't that 400Hz?

Reply:
https://github.com/ArduPilot/ardupilot/blob/70a759e001affce0e612cfbc591ad3bd58a3be37/libraries/AP_Vehicle/AP_Vehicle.cpp#L215

Reply:
Thanks, removed that commit